generators/app/templates/src/app/core/shell/__ionic._shell.component.ts

```diff
@@ -132,6 +132,20 @@ export class ShellComponent implements OnInit {
     route = route.firstChild;
     if (route && route.component === ShellComponent && route.firstChild) {
       route = route.firstChild;
```

Comment:
yes, that is the another bug I've refered.

You can use `<router-outlet>` instead of `<ion-nav>` to see the lazy module loaded correctly.

```html
<ion-split-panee>
</ion-split-panee>
<router-outlet></router-outlet>
```

Reply:
Ok that's it! But using `<router-outlet>` instead of `<ion-nav>` causes CSS issues with the split pane, toollbars and prevents using Ionic animations, so it's still not ideal 😕 

I'll merge the PR anyways, but I think I'll disable the lazy loading option prompt for Ionic until I find a proper fix for navigation.